src/jaguar.toit

```diff
@@ -282,22 +282,89 @@ handle_browser_request request/http.Request writer/http.ResponseWriter -> none:
   if path == "index.html":
     writer.headers.set "Content-Type" "text/html"
     writer.write """
-        <html>
-          <head>
-            <link rel="stylesheet" href="style.css">
-            <title>$device.name (Jaguar device)</title>
-          </head>
-          <body>
-            <h1>Jaguar: $device.name</h1>
-            <h2>Uptime: $(Duration --s=Time.monotonic_us / Duration.MICROSECONDS_PER_SECOND)</h2>
-            <h2>Toit SDK version: $vm_sdk_version</h2>
-            <h2 class=help>Run code on this device using <a href="https://github.com/toitlang/jaguar"><code>jag run</code></a></h2>
-            <h2 class=help>Monitor the serial port console using <a href="https://github.com/toitlang/jaguar"><code>jag monitor</code></a></h2>
-            <p>
-              <img src="$CHIP_IMAGE" alt="Picture of an embedded device" width=200 />
-            </p>
-          </body>
-        </html>
+<html>
```

Comment:
Toit tries to be smart about stripping the leading indentation. Maybe keep it indented, so it flows naturally with the source code?

Reply:
done [75986fc](https://github.com/toitlang/jaguar/pull/226/commits/75986fc47f932a95519700856d385a240149c0a9)